python/ray/data/dataset.py

```diff
@@ -323,7 +319,8 @@ def parse_filename(row: Dict[str, Any]) -> Dict[str, Any]:
             compute: The compute strategy, either None (default) to use Ray
                 tasks, ``ray.data.ActorPoolStrategy(size=n)`` to use a fixed-size actor
                 pool, or ``ray.data.ActorPoolStrategy(min_size=m, max_size=n)`` for an
-                autoscaling actor pool.
+                autoscaling actor pool. This argument is deprecated. Please use
+                ``concurrency`` argument instead.
```

Comment:
also update the docstring for the `fn` parameter on how it decides using tasks or actors? can just add doc link if that's too long.

Reply:
Updated to add a link above.

Reply:
Updated.